variables.tf

```
@@ -63,6 +63,12 @@ variable "log_config_enable" {
   default     = false
 }
 
+variable "static_ip_nat" {
```

What do you think of:

```suggestion
variable "nat_static_ip" {
```